app/views/admin/overview/index.html.erb

```diff
@@ -0,0 +1,63 @@
+<h2 class="crayons-title mb-3">Overview</h2>
+<div class="grid s:grid-cols-3 s:grid-rows-4 gap-3">
+  <article style="grid-column: span 2; grid-row: span 4;">
```

Comment:
I'd love to get rid of these inline styles, but I'm not sure if we have utility classes to handle these attributes. Does anyone know if we do?

Reply:
@ludwiczakpawel probably knows the answer to this 😃

Reply:
We do not have utility classes for these. We can either add these to the [generator](https://github.com/forem/forem/blob/master/app/assets/stylesheets/config/_generator.scss) (I'm happy to help) or move these particular styles to appropriate CSS file oooor just leave it as you have here. I do think it's ugly and naughty :D but maybe it's just not worth the effort IF we have any plans to revisit this view anytime soon?

Reply:
ohhh, wait a minute....

here's something you may find useful:

```
<h2 class="crayons-title mb-3">Overview</h2>
<div class="px-0 crayons-layout crayons-layout--2-cols crayons-layout--2-cols--1-2 crayons-layout--2-cols--inverted">
    <div class="crayons-layout__content">
        Left column
    </div>
    <div class="crayons-layout__sidebar-right">
        Right column
    </div>
</div>
```